psychopy/data.py

```diff
@@ -3464,8 +3464,8 @@ class QuestHandler(StairHandler):
         # create staircase object
         # trying to find out the point where subject's response is 50 / 50
         # if wanted to do a 2AFC then the defaults for pThreshold and gamma
-        # are good
-        staircase = data.QuestHandler(staircase._nextIntensity, 0.2,
+        # are good. As start value, we'll use 50% contrast, with SD = 20%
+        staircase = data.QuestHandler(0.5, 0.2,
```

Comment:
> https://github.com/psychopy/psychopy/blob/c564dbd61be91bed6ab47c50f9e9ae8547c2883b/psychopy/contrib/quest.py#L293
> 
> Definitely not 20%.

This is simply a normal distribution with SD equal to `startValSD` and mean `startVal`.